pkg/polylog/context_example_test.go

```diff
@@ -0,0 +1,43 @@
+package polylog_test
+
+import (
+	"context"
+	"os"
+
+	"github.com/rs/zerolog"
+
+	"github.com/pokt-network/poktroll/pkg/polylog"
+	"github.com/pokt-network/poktroll/pkg/polylog/polyzero"
+)
+
+func ExampleCtx() {
```

Comment:
```suggestion
// This function is for illustrative purposes only of how to use polylog in practice
func ExampleCtx() {
```

Reply:
Recall that this is an established convention ("testable examples"). The name of this function MUST be this in order for it to render next in the appropriate section of the godoc:

![image](https://github.com/pokt-network/poktroll/assets/600733/967715da-4ef3-4da3-bbd9-1e28fd68ad63)

Wdyt about making adding a TODO somewhere (code or issue) to include a primer on go testable examples and/or link to the [blog](https://go.dev/blog/examples) and [testing pkg docs](https://pkg.go.dev/testing#hdr-Examples) that talk about it.